### PR TITLE
fixes #21: use merkletreejs for getRootHash

### LINKED ISSUE #21: Implement `getRootHash`
## Description

Use all attributes `gameId`, `player`, and `score` to determine the getRootHash.

### PR DESCRIPTION
**Pull request by eshaan7**
Closes #21

### CODE REVIEW

File: rollup/stackr/machine.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { State, StateMachine } from "@stackr/sdk/machine";
-import { merklize } from "@stackr/sdk";
-import { solidityPacked } from "ethers";
+import { keccak256, solidityPackedKeccak256 } from "ethers";
+import MerkleTree from "merkletreejs";
```
Comment (aashutoshrathi) [inline rollup/stackr/machine.ts]:
```suggestion
import { MerkleTree } from "merkletreejs";
```

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Also, let's not use `merkletreejs` let's use our own `merklize` as it doesn't return good stateRoot on empty array.

File: rollup/stackr/machine.ts (hunk 1 above)
Comment (aashutoshrathi) [reply rollup/stackr/machine.ts]: this because it won't make runnable WASM

**Review by 0xRampey — APPROVED** [review]
lgtm

**Pull request merged**